src/screens/NotificationList.re

```diff
@@ -132,7 +164,15 @@ let make = (_children) => {
                         text
                         color
                         icon
-                        task=(notification##task)
+                        onClick=((_) => {
+                            if (Js.Nullable.test(notification##readDate)) {
+                                reduce((_) => ReadNotification(notification##id), ())
```

Comment:
I see that your objective here is to read the notifications. But what this `reduce` means?

Reply:
`reduce` is from React for Reason. It is made to dispatch an action that can work on the state (in the reducer).

Reply:
Oh, it is actually the `dispatch` function, i get it...